Content.Shared/DeadmanSwitch/DeadmanSwitchEvent.cs

```diff
@@ -0,0 +1,9 @@
+using Content.Shared.DoAfter;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.DeadmanSwitch;
+
+[Serializable, NetSerializable]
+public sealed partial class DeadmanSwitchDoAfterEvent : SimpleDoAfterEvent
+{
+}
```

Comment:
```suggestion
[Serializable, NetSerializable]
public sealed partial class DeadmanSwitchDoAfterEvent : SimpleDoAfterEvent;
```
Add documentation. Add new line at the end of the file. You might want to check if your IDE is configured correctly, this should be impossible to do wrong if it reads the .editorconfig correctly.